packages/framework/attributor/src/runtimeAttributorDataStoreChannel.ts

```diff
@@ -0,0 +1,286 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { TypedEventEmitter } from "@fluid-internal/client-utils";
+import { AttachState, IDeltaManager } from "@fluidframework/container-definitions/internal";
+import {
+	FluidObject,
+	IFluidHandle,
+	IRequest,
+	IResponse,
+} from "@fluidframework/core-interfaces";
+import type { IFluidHandleInternal } from "@fluidframework/core-interfaces/internal";
+import { assert, Deferred, unreachableCase } from "@fluidframework/core-utils/internal";
+import { FluidObjectHandle } from "@fluidframework/datastore/internal";
+import { IFluidDataStoreRuntimeEvents } from "@fluidframework/datastore-definitions/internal";
+import {
+	IDocumentMessage,
+	type ISnapshotTree,
+	ISequencedDocumentMessage,
+	IQuorumClients,
+} from "@fluidframework/driver-definitions/internal";
+import {
+	IGarbageCollectionData,
+	IFluidDataStoreChannel,
+	IFluidDataStoreContext,
+	IInboundSignalMessage,
+	VisibilityState,
+	type ISummaryTreeWithStats,
+	type ITelemetryContext,
+} from "@fluidframework/runtime-definitions/internal";
+import {
+	ITelemetryLoggerExt,
+	MonitoringContext,
+	raiseConnectedEvent,
+	createChildMonitoringContext,
+} from "@fluidframework/telemetry-utils/internal";
+
+import {
+	type IProvideRuntimeAttributor,
+	type IRuntimeAttributor,
+} from "./attributorContracts.js";
+import { RuntimeAttributor } from "./runtimeAttributor.js";
+
+/**
+ * Data store channel for the runtime attributor. This channel is responsible for storing and managing the
+ */
+export class RuntimeAttributorDataStoreChannel
+	extends TypedEventEmitter<IFluidDataStoreRuntimeEvents>
+	implements IFluidDataStoreChannel, IProvideRuntimeAttributor
```

Comment:
I don't think this needs to / should implement `IProvideRuntimeAttributor`.  The only way to get the Attributor is to use the alias, which returns the entry point not this channel.  Right?

Reply:
Removed.